custom_components/samsungtv_tizen/media_player.py

```diff
@@ -115,6 +123,8 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         hass.data[KNOWN_DEVICES_KEY] = known_devices
 
     uuid = None
+    show_channel_number = False
+    scan_app_http = True
```

Comment:
`show_channel_number` is already set as default `False` here: `vol.Optional(CONF_SHOW_CHANNEL_NR, default=False): cv.boolean,`

`scan_app_http` is also already set as default `True` here: `vol.Optional(CONF_SCAN_APP_HTTP, default=True): cv.boolean,`

So why are these 2 lines needed?